.github/workflows/build-workspace.yml

```diff
@@ -14,7 +14,7 @@ env:
 
 jobs:
   build-linux:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

Comment:
I downgraded to 20.04 because the resulting binaries cannot be run on a glibc version older than the one they were built with--see https://github.com/valadaptive/ntsc-rs/issues/215.

It seems the 20.04 runner image has been removed from Actions since the CI pipeline last ran--would it be possible to use 22.04 instead?